utils/run_test.sh

```diff
@@ -117,3 +131,60 @@ if [[ "$CART_TEST_MODE" =~ (memcheck|all) ]]; then
   rsync -rm --include="*/" --include="valgrind*xml" "--exclude=*" ${TESTDIR} ${RESULTS}
 
 fi
+
+CART_BASE=\${SL_PREFIX%/install*}
```

Comment:
(lint) Brace expansions and globs are literal in assignments. Quote it or use an array. [SC2125]
(lint) This { is literal. Check expression (missing ;/\n?) or quote it. [SC1083]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]